src/Maestro/Maestro.Web/Scripts/Set-TlsConfiguration.ps1

```diff
@@ -0,0 +1,738 @@
+#***************************************************************************************************************
```

Comment:
Is this file copied from somewhere? It _feels_ like it is, since it's so large and has some... unusual stuff inside it. If it is, I don't really want to spend a ton of time reviewing it.

Reply:
It is coming from https://microsoft.sharepoint.com/teams/TLS12Everywhere

Reply:
I didn't want to change it and add docs about where it came from because it is signed in the bottom

Reply:
Seems fine then. The fact that we can't modify the script is... unfortunate, but as long as we make sure the commit message has the URL in it, that should be enough to find it, I hope.

Reply:
I put it in the PR description above and will include it in the commit message